test/client-test.js

```diff
@@ -97,6 +98,68 @@ describe('SOAP Client', function() {
     });
   });
 
+  describe('Create a client from a wsdl preloaded in the options', function() {
+
+    var defaultNamespaceWsdl
+
+    beforeEach(function(done) {
+
+      // Read the contents of the WSDL from the file system
+      fs.readFile(__dirname + '/wsdl/default_namespace.wsdl', 'utf8', function (err, defaultNamespaceWsdlContents) {
+        if (err) {
+          done(err)
+        } else {
+
+          var options = {
+            WSDL_CACHE: {}
+          }
+          // Create the initial wsdl directly
+          defaultNamespaceWsdl = new WSDL(defaultNamespaceWsdlContents, undefined, options)
```

Comment:
nitpick: can we leave `options` out or just use `{}` here?

Reply:
Certainly. Done.